x-pack/plugins/security_solution/public/users/pages/navigation/user_risk_score_tab_body.tsx

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useMemo } from 'react';
+import { noop } from 'lodash/fp';
+
+import { UsersComponentsQueryProps } from './types';
+import { manageQuery } from '../../../common/components/page/manage_query';
+import { useDeepEqualSelector } from '../../../common/hooks/use_selector';
+import { State } from '../../../common/store';
+
+import { UserRiskScoreTable } from '../../components/user_risk_score_table';
+import { usersSelectors } from '../../store';
+import {
+  UserRiskScoreQueryId,
+  useUserRiskScore,
+  useUserRiskScoreKpi,
+} from '../../../risk_score/containers';
+
+const UserRiskScoreTableManage = manageQuery(UserRiskScoreTable);
+
+export const UserRiskScoreQueryTabBody = ({
+  filterQuery,
+  skip,
+  setQuery,
+  type,
+  deleteQuery,
+}: UsersComponentsQueryProps) => {
+  const getUserRiskScoreSelector = useMemo(() => usersSelectors.userRiskScoreSelector(), []);
```

Comment:
do redux selectors need `useMemo`?

Reply:
We do use this pattern on most selectors that call `createSelector`. I took a look into the documentation and they have a similar example:
https://react-redux.js.org/api/hooks#useselector-examples 
<img width="993" alt="Screenshot 2022-03-04 at 10 08 08" src="https://user-images.githubusercontent.com/1490444/156733732-91b51c83-60cd-446e-b1ec-ae7d9ba055ef.png">

Going deeper into the docs I found a better explanation:
https://github.com/reduxjs/reselect#q-can-i-share-a-selector-across-multiple-component-instances


> Q: Can I share a selector across multiple component instances?
> A: Yes, although it requires some planning.
> 
> As of Reselect 4.1, you can create a selector with a cache size greater than one by passing in a maxSize option under memoizeOptions for use with the built-in defaultMemoize.
> 
> Otherwise, selectors created using createSelector only have a cache size of one. This can make them unsuitable for sharing across multiple instances if the arguments to the selector are different for each instance of the component. There are a couple of ways to get around this:
> 
> Create a factory function which returns a new selector for each instance of the component. This can be called in a React component inside the useMemo hook to generate a unique selector instance per component.
> Create a custom selector with a cache size greater than one using createSelectorCreator
>